website/content/partials/known-issues/duplicate-identity-group.mdx

```diff
@@ -0,0 +1,12 @@
+### Duplicate identity groups created when concurrent requests sent to the primary and PR secondary cluster
+
+#### Affected versions
+- All vault versions v1.17.2+ent, v1.16.6+ent, v1.15.12+ent and below
```

Comment:
Functionally this makes sense, but we should specify that this is restricted to Enterprise only, and technically only the versions where Performance Replication was an actual feature. Maybe something like `All Enterprise versions from 0.7.0 up`, but that version is a bit fuzzy and would need verification.

Reply:
good call, updated!